src/Polly.Testing/ResilienceStrategyExtensions.cs

```diff
@@ -56,6 +56,11 @@ private static Type GetStrategyType<T>(ResilienceStrategy strategy)
             return bridge.Strategy.GetType();
         }
 
+        if (strategy is NonReactiveResilienceStrategyBridge bridge2)
```

Comment:
Can we call this `nonreactiveBridge` rather than `bridge2`?

Reply:
Sure!